tests/common/plugins/sanity_check/checks.py

```diff
@@ -154,6 +155,78 @@ def check_dbmemory(dut):
     logger.info("Done checking database memory")
     return check_result
 
+def check_monit_services_status(check_result, monit_services_status):
+    """
+    @summary: Check whether each type of service which was monitored by Monit was in correct status or not.
+              If a service was in "Not monitored" status, sanity check will skip it since this service
+              was temporarily set to not be monitored by Monit.
+    """
+    check_result["services_status"] = {}
+    for service_name, service_info in monit_services_status.items():
+        check_result["services_status"].update({service_name: service_info["service_status"]})
+        if service_info["service_status"] == "Not monitored":
+            continue
+        if ((service_info["service_type"] == "Filesystem" and service_info["service_status"] != "Accessible")
+            or (service_info["service_type"] == "Process" and service_info["service_status"] != "Running")
+            or (service_info["service_type"] == "Program" and service_info["service_status"] != "Status ok")):
```

Comment:
You still haven't allowed 'Not monitored', am I missing something elsewhere?

Reply:
I skip testing the service which was in Not monitored status at line 167 ~ 168.